app/controllers/businessmatching/ConfirmPostCodeController.scala

```diff
@@ -43,7 +45,7 @@ class ConfirmPostCodeController @Inject()(authAction: AuthAction,
 
   def updateReviewDetails(reviewDetails: Option[ReviewDetails], postCodeModel: ConfirmPostcode): Option[models.businesscustomer.ReviewDetails] = {
     reviewDetails.fold[Option[ReviewDetails]](None) { dtls =>
-      val updatedAddr = dtls.businessAddress.copy(postcode = Some(postCodeModel.postCode))
+      val updatedAddr = dtls.businessAddress.copy(postcode = Some(postCodeModel.postCode), country = Country("United Kingdom", "GB"))
```

Comment:
this was changed because previously you could have in business details address with empty country Country("","") and valid UK postcode. This will now update not only postcode but also will set the country to UK if postcode will pass the validation